fastapi-app/routers/recipe.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any
+
+
+from sqlalchemy.orm import Session
+from fastapi import APIRouter, Depends
+
+
+from dependencies import get_db
+
+from services.recipe import service_create_new_recipe, \
+    service_get_recipe_by_name, service_get_recipes_with_max_ingredients
+
+from schemas.recipe import Recipe as RecipeSchema, RecipeCreate
+
+router = APIRouter(prefix='/recipe')
+
+
+@router.post('/create', response_model=RecipeSchema)
+def create_new_recipe(
+        recipe: RecipeCreate, db: Session = Depends(get_db)
+) -> Any:
+    # recipe - contains name, preparation and dict[ingredient]=quantity
+
+    response = service_create_new_recipe(recipe, db)
```

Comment:
If you decide to stick with variables instead of `return func()` you should have consistency, either make it all `response` or make it all contextual with the objects that it will return.
If the case is that `service_create_new_recipe` doesn't return anything, it should. Whenever you create a new resource, it should be sent back to the user as a response on creation.